ansible/Makefile

```diff
@@ -19,9 +19,12 @@ apply:
 	cd ../$(TERRAFORM_DIR) && terraform apply && terraform state pull > terraform.tfstate
 
 .PHONY: ansible
-ansible: install_roles init plan apply install_jenkins
+ansible: install_roles init plan apply install_jenkins harden_server
```

Comment:
Presumably we'll do this all through Packer sooner or later, but this works for now.